modules/nw-osp-pod-connections-ipv6.adoc

```diff
@@ -0,0 +1,32 @@
+// Module included in the following assemblies:
+//
+// * post_installation_configuration/network-configuration.adoc
+
+:_content-type: PROCEDURE
+[id="nw-osp-pod-connections-ipv6_{context}"]
+= Enabling IPv6 connectivity to pods on {rh-openstack}
+
+To enable IPv6 connectivity between pods that have additional networks that are on different nodes, disable port security for the IPv6 port of the server. By disabling port security, you can add allowed address pairs with an IP and MAC address in the IPv6 port of the server whenever a new pod is created.
```

Comment:
> By disabling port security, you can add allowed address pairs with an IP and MAC address in the IPv6 port of the server whenever a new pod is created.

Not exactly, the interest of disabling port security in this case is to not have to deal with allowed address pairs at all because it gets in the way. See https://github.com/openshift/installer/pull/6435#discussion_r989177706 for more context.

Reply:
```suggestion
To enable IPv6 connectivity between pods that have additional networks that are on different nodes, disable port security for the IPv6 port of the server. Disabling port security prevents the creation of allowed address pairs with IP and MAC addresses on the server's IPv6 port whenever a new pod is created.
```